apps/spruce/src/constants/requesters.ts

```diff
@@ -10,6 +10,16 @@ enum Requester {
   Trigger = "trigger_request",
 }
 
+const commitRequesters = [
```

Comment:
nit. can we rename this to `mainlineRequesters` or `mainlineCommitRequesters`. So we maintain consistent verbiage.